### PR TITLE
Issue5

### LINKED ISSUE #5: As a user, I want to control the arrow from buttons below the canvas
> I'll try to use a more agile-like title from now on

As a user I want to control the arrow by clicking buttons positioned horizontally below the canvas.

The buttons should be represented in a frame, called "Mission Control"

### PR DESCRIPTION
**Pull request by Steppico**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,11 +12,14 @@
 
 <body>
     <h1>a-MAZE-ing</h1>
-    <button class="moveForward">⬇️</button>
-    <button class="rotate">↩️</button>
-    <button class="execute">✅</button>
+    <h3 class="missionControl-title">Mission Control</h3>
```
Comment (zero4994) [inline src/index.html:15]: lol why mission control?

File: src/index.html [hunk 2]
```diff
@@ -12,11 +12,14 @@
...
+    <div class="missionControl">
+        <button class="moveForward">⬇️</button>
```
Comment (zero4994) [inline src/index.html:17]: Can this use something like font awesome to make the icons a little bigger? Its not required tho

File: src/index.html (hunk 1 above)
Comment (Steppico) [reply src/index.html:15]: good question lmao I don't know I put it as a placeholder and it kinda looked nice, so it stuck :D

Comment (zero4994) [reply src/index.html:15]: Haha nice

File: src/index.html (hunk 2 above)
Comment (Steppico) [reply src/index.html:17]: 🤯 I didn't know about awesome. Let's add them! I'll share the new icons with you first, then I'll implement them. Thanks!

**Review by zero4994 — APPROVED** [review]
Good work 👍

File: src/index.html (hunk 2 above)
Comment (zero4994) [reply src/index.html:17]: Awesome, not required for this PR, don't wanna hold you back

**Comment by Steppico** [comment]
Merged, let's create a new issue for better icons! 😄

**Pull request merged**